presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveLocationService.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.hive;
+
+import org.apache.hadoop.fs.Path;
+import org.testng.annotations.Test;
+
+import static org.mockito.Mockito.mock;
+import static org.testng.Assert.assertEquals;
+
+public class TestHiveLocationService
+{
+    @Test
+    public void testGetTableWriteInfo_append()
```

Comment:
Nit: use an underscore violates the standard Java method naming convention. Name this `testGetTableWriteInfoAppend()`. I do agree it's more readable in this case, but it looks out of place in our code base.